cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -100,6 +105,32 @@ private void checkServiceResources(Set<String> problemKeys) {
     }
   }
 
+  private void checkBindAddresses() {
+    // check if service bind addresses are loopback addresses
+    ImmutableSet.Builder<String> builder = ImmutableSet.<String>builder()
```

Comment:
It can work for more than 6:

![image](https://cloud.githubusercontent.com/assets/2440977/16254948/e5c9459c-37f9-11e6-9e07-25f63ad69cc1.png)